src/main/java/fun/epak/pak/infrastructure/OtherUserProfileData.java

```diff
@@ -0,0 +1,18 @@
+package fun.epak.pak.infrastructure;
+
+import fun.epak.pak.model.user.User;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+public class OtherUserProfileData {
+    private long id;
+    private String username;
+    private String imageNameAddress;
+
+    public static OtherUserProfileData of(User user, String path) {
+        return new OtherUserProfileData(user.getId(), user.getUsername(), path);
```

Comment:
z Usera czytamy tylko 2 pola. Zastanowiłbym się czy nie lepiej zbudować ten obiekt builderem w serwisie

Reply:
Będziemy doczytywać później posty jego wiec wolałem to zrobić w ten sposób